packages/api/src/routes-v1/nfts-list.js

```diff
@@ -0,0 +1,33 @@
+import { JSONResponse } from '../utils/json-response.js'
+import { validate } from '../utils/auth-v1.js'
+import { toNFTResponse } from '../utils/db-client.js'
+
+/** @type {import('../utils/router.js').Handler} */
+export async function nftListV1(event, ctx) {
+  const { user, db } = await validate(event, ctx)
+  const options = {
+    limit: 10,
+    before: new Date().toISOString(),
+  }
+  const { searchParams } = new URL(event.request.url)
+
+  const limit = searchParams.get('limit')
```

Comment:
Needs a max. Currently is max 100, but we could now increase to 1,000 WDYT?

Reply:
added task here https://github.com/ipfs-shipyard/nft.storage/issues/331